docs/podman-volume-prune.1.md

```diff
@@ -0,0 +1,38 @@
+% podman-volume-prune(1)
+
+## NAME
+podman\-volume\-prune - Remove all unused volumes
+
+## SYNOPSIS
+**podman volume rm** [*options*]
+
+## DESCRIPTION
+
+Deletes all unused volumes. You will be prompted to confirm the removal of all the
```

Comment:
You did this in code comments too.  That is flipped between using the word remove and delete.  I'd pick one and stick with it throughout.  For prune I'd personally stick with remove, but delete works too.  Your call.

Reply:
Stuck with remove, and fixed in all places :)